tests/macsec/test_macsec.py

```diff
@@ -180,7 +180,7 @@ class TestFaultHandling():
     MKA_TIMEOUT = 6
     LACP_TIMEOUT = 90
 
-    def test_link_flap(self, duthost, ctrl_links):
+    def test_link_flap(self, duthost, ctrl_links, tbinfo):
```

Comment:
As previous comments, we don't really need `tbinfo` at this function.

Reply:
Done for all, Remove all the tdInfo parameter added, did that earlier for the portchannel skip which I later removed